Curtains/Curtains.Domain/Models/MarketingInfoModel.cs

```diff
@@ -25,5 +25,12 @@ public class MarketingInfoModel : BaseEntity
         /// </summary>
         public bool IsPopular { get; set; }
         #endregion
+
+        #region NavigationProperties
+        /// <summary>
+        ///  This property represents ProductImage.
+        /// </summary>
+        public virtual ProductImageModel ProductImage { get; set; }
+        #endregion
```

Comment:
Why did you add this property?

Reply:
it shows the product image it is connected with